### PR TITLE
update: docs reflect update to docker-compose

### PR DESCRIPTION
**Pull request by JamesKunstle**
deprecates scripts/launch_dev.sh in favor of compose strategy.

Signed-off-by: James Kunstle <jkunstle@bu.edu>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,37 +94,32 @@ To get these credentials please contact one of the maintainers of this project o
...
+The webserver, worker-pool, and cache containers communicate with one another via docker network.
 
-Please run the following to launch your app:
+All of the build/tear-down is done with docker-compose.
+
+To start the application, run at the top-level of the 8Knot directory:
 
 ```bash
-bash scripts/launch_dev.sh
+docker compose up --build
```
Comment (cdolfi) [inline README.md:108]: Can you add somewhere in the documentation about the install requirements for docker and docker compose?

**Review by cdolfi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JamesKunstle) [reply README.md:118]: updated above in the "runtime" section

**Comment by JamesKunstle** [comment]
@cdolfi updated whenever ready

File: README.md [hunk 2]
```diff
@@ -94,36 +94,41 @@ To get these credentials please contact one of the maintainers of this project o
...
-see if this is the case by running:
+To stop the application, run:
 
 ```bash
-ps aux | grep redis
+ctrl-c
 ```
 
-Kill any existing redis-server instance by ID and re-run 'launch_dev.sh'.
+To clean up the stopped containers, run:
```
Comment (cdolfi) [inline README.md:127]: Is this still the case?

Comment (JamesKunstle) [reply README.md:127]: no, need to update this, good catch

File: README.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ We've tried to make it as easy as possible to go from fork/clone to a running in
 
 ### Credentials
```
Comment (cdolfi) [inline README.md:76]: Not sure exactly where we should add this info, but we now have a small demo augur instance for people to use. 
Credentials here: 
{
    "connection_string": "sqlite:///:memory:",
    "database": "astros",
    "host": "chaoss.tv",
    "password": "!xpk98T6?bK",
    "port": 5432,
    "schema": "augur_data",
    "user": "eightknot",
    "user_type": "read_only"
}

**Review by cdolfi — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (JamesKunstle) [reply README.md:76]: okay added credentials

**Review by cdolfi — APPROVED** [review]

**Pull request merged**